src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas3/Cas3DomainEventService.kt

```diff
@@ -231,6 +233,34 @@ class Cas3DomainEventService(
     )
   }
 
+  @Transactional
+  fun saveBedspaceUnarchiveEvent(bedspace: BedEntity, currentStartDate: LocalDate, currentEndDate: LocalDate) {
+    val user = userService.getUserForRequest()
+    val domainEvent = cas3DomainEventBuilder.getBedspaceUnarchiveEvent(bedspace, currentStartDate, currentEndDate, user)
+
+    domainEventRepository.save(
+      DomainEventEntity(
+        id = domainEvent.id,
+        applicationId = null,
+        assessmentId = null,
+        bookingId = null,
+        crn = domainEvent.crn,
+        nomsNumber = null,
+        type = enumTypeFromDataType(domainEvent.data::class),
+        occurredAt = domainEvent.occurredAt.atOffset(ZoneOffset.UTC),
```

Comment:
Why is one UTC and the other one has no timezone? @besscerule

Reply:
It's because under the hood we store the date as UTC anywhere and when running in docker it's UTC by default anyway. But because UTC is not used everywhere in the code, locally some tests fail as BST is 1 hr ahead of UTC atm.